lib/src/list_edit_matrix.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:math';
+
+import 'package:collection/collection.dart';
+
+/// Utility class for diffing lists using the Wagner–Fischer algorithm.
+/// It creates a matrix of operations which need to be applied to the old list
+/// to recreate the new one.
```

Comment:
I think you put these comments in front of the wrong type.

Reply:
👍